DS13/code/game/movietitles.dm

```diff
@@ -0,0 +1,38 @@
+//Makes end credits happen after round end!
+//Inspired by baystation, optimized for /tg/
+
+/datum/asset/simple/credits
+	assets = list("credits.gif"	= 'DS13/icons/stars2.gif')
+
+/proc/names2credits()
+	var/credits = "Starring the crew of [station_name()]<br>"
+	for(var/datum/data/record/t in GLOB.data_core.general)
+		var/name = t.fields["name"]
+		var/rank = t.fields["rank"]
+		credits += "[rank] [name]<br>"
+	credits += "Directed by: Kmc2000 <br> Executive producer: Francinum <br> Technical supervisor: Bass-ic <br>"
+	return credits
+
+/proc/start_credits_global()
+	var/text = names2credits()
+	SEND_SOUND(world, 'DS13/sound/effects/endcredits.ogg')
+	for(var/client/C in GLOB.clients)
+		if(C.mob)
+			C.mob.roll_credits(text)
+
+/mob/proc/roll_credits(var/credits_list)
+	var/client/user = src
+	var/datum/asset/assets = get_asset_datum(/datum/asset/simple/credits)
+	assets.send(user)
+	var/thing = credits_list
+	var/credits = "\
+	<body bgcolor='#000000'>\
+	<body background='credits.gif'>\
```

Comment:
Move these css tags to the css file,having more than one body is a violation of HTML standards

Reply:
Couldn't figure out how to get those to work 🙌